client/package.json

```diff
@@ -5,6 +5,12 @@
   "dependencies": {
     "@fortawesome/fontawesome-common-types": "^0.2.26",
     "@fortawesome/free-solid-svg-icons": "^5.12.0",
+    "@storybook/addon-actions": "^6.1.15",
+    "@storybook/addon-essentials": "^6.0.22",
+    "@storybook/addon-links": "^6.0.22",
+    "@storybook/node-logger": "^6.0.22",
+    "@storybook/preset-create-react-app": "^3.1.4",
+    "@storybook/react": "^6.1.15",
```

Comment:
I still get the error :(
![image](https://user-images.githubusercontent.com/17876556/106139279-581fe600-61c1-11eb-91d4-6339d8413d57.png)

Reply:
I realised while falling asleep last night that I forgot to run `yarn install`... Will try again tonight

Reply:
All good, I tested it again and it worked fine for me!

Reply:
Still not working for me :/ I'm going to assume this is on my end and look into it later.